Gems/ROS2/Code/Include/ROS2/Lidar/LidarRaycasterBus.h

```diff
@@ -153,9 +153,8 @@ namespace ROS2
         }
 
         //! Configures Layer ignoring parameters
-        //! @param ignoreLayer Should a specified collision layer be ignored?
-        //! @param layerIndex Index of collision layer to be ignored.
-        virtual void ConfigureLayerIgnoring([[maybe_unused]] bool ignoreLayer, [[maybe_unused]] AZ::u32 layerIndex)
+        //! @param layerIndices Indices of collision layers to be ignored.
+        virtual void ConfigureLayerIgnoring([[maybe_unused]] const AZStd::vector<AZ::u32>& layerIndices)
```

Comment:
Naming suggestion: 
SetIgnoredCollisionLayers

Reply:
This follows the naming convention of other Lidar raycast settings (ConfigureRayRange, ConfigureRaycastResultFlags, ...)

Reply:
Changed the name to ConfigureIgnoredCollisionLayers, as suggested